### PR TITLE
Final changelog update for 0.E

### PR DESCRIPTION
**Pull request by kevingranade**
SUMMARY: None

Final changelog, I AM interested in feedback on changelog copyediting and highlighted feature wording.

### CODE REVIEW

File: data/changelog.txt [hunk 1]
```diff
@@ -1,9 +1,27 @@
...
+
+## Highlights:
+Long distance automove feature for walking, driving and boating.
+Extensive bugfixes to inter-level interactivity, on by default.
+Riding animals and animal-pulled vehicles.
+More flexible Basecamp construction options.
+Default starting season changed to spring for better survivability.
+Time advancement is rationalized, a turn is now one second.
+Extensive river and lake systems, and boat support for navigating them.
+Expanded NPC usefullness and interactivity.
```
Comment (reinerh) [inline data/changelog.txt]: typo: usefulness

File: data/changelog.txt [hunk 2]
```diff
@@ -1,9 +1,27 @@
-# 0.E (2020-03-21)
+# 0.E (Ellison)
+
+## Highlights:
+Long distance automove feature for walking, driving and boating.
+Extensive bugfixes to inter-level interactivity, on by default.
+Riding animals and animal-pulled vehicles.
+More flexible Basecamp construction options.
+Default starting season changed to spring for better survivability.
```
Comment (esotericist) [inline data/changelog.txt]:
```suggestion
Default starting date changed to mid-spring for better survivability.
```
it was always spring, just in the form of "winter only technically ended"

File: data/changelog.txt [hunk 3]
```diff
@@ -495,6 +513,7 @@ Adjust MRE caloric value to reflect real MREs.
 Lower max stat cap in character creation to 14.
 Overhaul archery balance.
```
Comment (Insanegame27) [inline data/changelog.txt:514]: Archery overhaul completely breaks archery craftable progression and makes you rely on getting extraordinarily lucky with world spawns to progress to lategame content

Comment (GhostEcho) [reply data/changelog.txt:514]: Gotta find a hunting store Insane :^)

**Pull request merged**